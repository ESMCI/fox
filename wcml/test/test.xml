<?xml version=".0" encoding="UTF-8"?>
<cml convention="FoX_wcml-2.0" fileId="test.xml" version="2.4"
  xmlns="http:www.xml-cml.orgschema"
  xmlns:xsd="http:www.w3.org200XMLSchema"
  xmlns:fpx="http:www.uszla.me.ukfpx"
  xmlns:dc="http:purl.orgdcelements.title"
  xmlns:units="http:www.uszla.me.ukFoXunits"
  xmlns:cmlUnits="http:www.xml-cml.orgunitsunits"
  xmlns:siUnits="http:www.xml-cml.orgunitssiUnits"
  xmlns:atomicUnits="http:www.xml-cml.orgunitsatomic">
  <property title="name">
    <array size="2" dataType="xsd:boolean">true false<array>
  <property>
<cml>
