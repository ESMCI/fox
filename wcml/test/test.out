./test.sh: line 8: ././test_cml_*.exe: No such file or directory
